components/com_finder/tmpl/search/default_result.php

```diff
@@ -72,6 +89,14 @@
 			<?php echo $this->result->title; ?>
 		<?php endif; ?>
 	</p>
+	<?php if ($show_image && isset($image)) : ?>
+        <figure class="<?php echo htmlspecialchars($imageclass, ENT_COMPAT, 'UTF-8'); ?> result__image">
+            <img src="<?php echo htmlspecialchars($image->url, ENT_COMPAT, 'UTF-8'); ?>"
+			<?php echo $alt; ?>
+			<?php echo $extraAttr; ?>
+            />
+        </figure>
```

Comment:
```suggestion
		<figure class="<?php echo htmlspecialchars($imageclass, ENT_COMPAT, 'UTF-8'); ?> result__image">
			<img src="<?php echo htmlspecialchars($image->url, ENT_COMPAT, 'UTF-8'); ?>"
				<?php echo $alt; ?>
				<?php echo $extraAttr; ?>
			/>
		</figure>
```
CS: Use tabs and not spaces to indent.